contracts/proxies/CurrencyConvertor.sol

```diff
@@ -201,16 +216,14 @@ contract CurrencyConvertor is BaseRelayRecipient {
     require(success, string(returndata));
 
     // Deposit change in balance of USDC to the L2 exchange account of the sender.
-    uint256 usdcBalanceChange = USDC_ADDRESS.balanceOf(address(this)) - originalUsdcBalance;
```

Comment:
Getting rid of `usdcBalanacheChange` isn't good. You can fix `Stack too deep` by using the following above:

```solidity
    // Swap token
    // Limit variables on the stack to avoid “Stack too deep” error.
    {
      (bool success, bytes memory returndata) = exchange.call(data);
      require(success, string(returndata));
    }
```